### DISCUSSION TITLE
Maybe host submissions on PIFOP?

### DISCUSSION THREAD

**Discussion by nidoro** [ambros-gleixner/MIPcc23#5] [Ideas]
Hi,

I'm Davi from [PIFOP](https://pifop.com), an online IDE for mathematical programming. I would like to offer our platform for free as a centralized place where all the competition submissions can be viewed, executed, compared, etc, so there is an easy and transparent way for everyone to see and verify everyone else's submission.

Aside from mathematical programs, PIFOP can also be used to compile/run C/C++, Python, Bash and virtually any other interpreter/compiler of any language, it is just a matter of configuring the [optimization server](https://pifop.com/help/opt_server_intro.html) where the submissions will be executed. Then the moderators and competitors would be able to see and run the models directly from the browser.

If you guys like the idea, let me know. It is good that we have a few months ahead of us so we have time to work out the details.

Have a nice weekend!

**Comment by ambros-gleixner** [comment]
Hi Davi, we will discuss this among the committtee, thank you for the suggestion!

**Reply by nidoro** [reply]
Ok! I've also sent you my email. If you have any questions, let me know.
